tests/datasource/fluent/test_contexts.py

```diff
@@ -149,6 +149,31 @@ def test_context_add_or_update_datasource(
     )
 
 
+@pytest.mark.cloud
+def test_cloud_add_or_update_datasource_kw_vs_positional(
+    cloud_api_fake: RequestsMock,
+    empty_cloud_context_fluent: CloudDataContext,
+    taxi_data_samples_dir: pathlib.Path,
+):
+    name: str = "kw_vs_positional_test"
+
+    datasource1 = empty_cloud_context_fluent.sources.add_pandas_filesystem(
+        name=name, base_directory=taxi_data_samples_dir
+    )
+
+    # pass name as keyword arg
+    datasource2 = empty_cloud_context_fluent.sources.add_or_update_pandas_filesystem(
+        name=name, base_directory=taxi_data_samples_dir
+    )
+
+    # pass name as positional arg
+    datasource3 = empty_cloud_context_fluent.sources.add_or_update_pandas_filesystem(
+        name, base_directory=taxi_data_samples_dir
+    )
+
+    assert datasource1 == datasource2 == datasource3
```

Comment:
makes total sense but I don't think I've ever use two equality expressions in a statement haha

Reply:
Can do this with other comparison operators too
```
assert 10 < 15 < 20
```

https://note.nkmk.me/en/python-chain-comparisons/
https://mathspp.com/blog/pydonts/chaining-comparison-operators